src/js/actions/history.js

```diff
@@ -104,10 +105,13 @@ define(function (require, exports) {
                 });
             })
             .then(function () {
-                var currentDocument = this.flux.store("application").getCurrentDocument();
+                var document = this.flux.store("application").getCurrentDocument(),
```

Comment:
That catch clause (or, at least, the comment therein) makes absolutely no sense to me.  Assuming it was me that wrote it ... maybe I meant "no targetLayers property means no layer is selected"?.

Oh, looks like it was probably copied from actions/layers.js : 458.

I'll fix the comment ... or experiment with removing the `catch` completely ... which seems less safe, but 'tis the season.